scripts/utils/internals.js

```diff
@@ -214,8 +214,17 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
     console.log("    Gas costs for " + subject + ": " + receipt.gasUsed)
   }
 
+  const getMasterCopy = async function(safeAddress) {
+    return new Promise(function(resolve, reject) {
+      web3.eth.getStorageAt(safeAddress, 0, (err, resp) => {
+        if (err) return reject(err)
+        resolve("0x" + resp.slice(26))
+      })
+    })
+  }
```

Comment:
I believe this might not be needed with our current version of web3, you might be able to just use:
```
const getMasterCopy = async function(safeAddress) {
  const resp = await web3.eth.getStorageAt(safeAddress, 0)
  return "0x" + resp.slice(26)
}
```